commcare_connect/data_export/views.py

```
@@ -250,7 +250,10 @@ def create(self, request, *args, **kwargs):
             item["opportunity"] = self.opportunity
             item["organization"] = self.opportunity.organization
             pk = item.pop("id", None)
-            obj, created = LabsRecord.objects.update_or_create(defaults=item, **{"id": pk})
+            if pk:
+                obj, created = LabsRecord.objects.update_or_create(defaults=item, id=pk)
```

Ruff (0.14.4)

Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)